cg/models/demultiplex/run_parameters.py

```diff
@@ -299,3 +311,13 @@ def get_read_1_cycles(self) -> int:
     def get_read_2_cycles(self) -> int:
         """Return the number of cycles in the second read."""
         return self._read_parser.get(RunParametersXMLNodes.READ_2_NOVASEQ_X)
+
+    def get_flow_cell_model(self) -> str:
+        """Return flow cell model."""
+        consumable_infos = self.tree.findall(".//ConsumableInfo")
+        for consumable_info in consumable_infos:
+            type_element = consumable_info.find("Type")
+            if type_element is not None and type_element.text == "FlowCell":
+                name_element = consumable_info.find("Name") or consumable_info.find("Mode")
```

Comment:
I think the `or consumable_info.find("Mode")` is not needed, I think it will always be `"Name"`
```suggestion
                name_element: ElementTree.Element | None = consumable_info.find("Name")
```

Reply:
we could simplify all these type hints by importing:
```python
from xml.etree.ElementTree import Element
```